pyqtgraph/graphicsItems/DateAxisItem.py

```diff
@@ -168,9 +168,16 @@ def tickValues(self, minVal, maxVal, minSpc):
             # reposition tick labels to local time coordinates
             ticks += self.utcOffset
             # remove any ticks that were present in higher levels
-            tick_list = [x for x in ticks.tolist() if x not in allTicks]
-            allTicks.extend(tick_list)
-            valueSpecs.append((spec.spacing, tick_list))
+            # we assume here that if the difference between a tick value and a previously seen tick value
+            # is less than min-spacing/100, then they are 'equal' and we can ignore the new tick.
+            ticks = np.array(ticks)
```

Comment:
I suppose this line is redundant since `ticks` is already an ndarray, something that wasn't obvious to me at the time.

Reply:
You're right, I updated it.